.travis.yml

```diff
@@ -45,10 +45,14 @@ before_script:
 - mkdir /tmp/build
 - travis_retry npm install
 - npm run lint
+- travis_retry meteor build --headless /tmp/build
```

Comment:
if you don't pass the `--directory` parameter, it will only create a gzipped bundle.. so the folder `/tmp/build-test/bundle/programs/server` will not exists